test/integration/isolated/ingress_test.go

```diff
@@ -196,3 +202,153 @@ func TestIngressGRPC(t *testing.T) {
+							// We'll use this annotation to test that modifications are allowed on KongServiceFacade
+							// level. This should make Kong Service created from this KongServiceFacade always
+							// return a PNG image.
+							annotations.AnnotationPrefix + annotations.PathKey: "/image/png",
+							annotations.IngressClassKey:                        ingressClass,
+						},
+					},
+					Spec: incubatorv1alpha1.KongServiceFacadeSpec{
+						Backend: incubatorv1alpha1.KongServiceFacadeBackend{
+							Name: service.Name,
+							Port: test.HTTPBinPort,
+						},
+					},
+				},
+			}
+			clients := GetFromCtxForT[*clientset.Clientset](ctx, t)
+			for _, serviceFacade := range serviceFacades {
+				serviceFacade, err = clients.IncubatorV1alpha1().KongServiceFacades(namespace).Create(ctx, serviceFacade, metav1.CreateOptions{})
+				require.NoError(t, err)
+				cleaner.Add(serviceFacade)
+			}
+
+			t.Log("configuring Ingress")
+			ingress := builder.NewIngress(uuid.NewString(), ingressClass).WithRules(
+				netv1.IngressRule{
+					IngressRuleValue: netv1.IngressRuleValue{
+						HTTP: &netv1.HTTPIngressRuleValue{
+							Paths: []netv1.HTTPIngressPath{
+								{
+									Path:     jpegIngressPath,
+									PathType: lo.ToPtr(netv1.PathTypePrefix),
+									Backend: netv1.IngressBackend{
+										Resource: &corev1.TypedLocalObjectReference{
+											APIGroup: lo.ToPtr(incubatorv1alpha1.SchemeGroupVersion.Group),
+											Kind:     incubatorv1alpha1.KongServiceFacadeKind,
+											Name:     serviceFacades[0].Name,
+										},
+									},
+								},
+								{
+									Path:     pngIngressPath,
+									PathType: lo.ToPtr(netv1.PathTypePrefix),
+									Backend: netv1.IngressBackend{
+										Resource: &corev1.TypedLocalObjectReference{
+											APIGroup: lo.ToPtr(incubatorv1alpha1.SchemeGroupVersion.Group),
+											Kind:     incubatorv1alpha1.KongServiceFacadeKind,
+											Name:     serviceFacades[1].Name,
+										},
+									},
+								},
+							},
+						},
+					},
+				}).
+				WithAnnotations(map[string]string{
+					annotations.AnnotationPrefix + annotations.StripPathKey: "true",
+				}).Build()
+			ingress, err = cluster.Client().NetworkingV1().Ingresses(namespace).Create(ctx, ingress, metav1.CreateOptions{})
+			require.NoError(t, err)
+			cleaner.Add(ingress)
+
+			return ctx
+		}).
+		Assess("KongServiceFacades annotations work", func(ctx context.Context, t *testing.T, c *envconf.Config) context.Context {
+			proxyURL := GetProxyURLFromCtx(ctx)
+			expectContentType := func(path, expectedMagicNumber string) {
+				t.Logf("asserting %s path returns expected image", path)
+				helpers.EventuallyGETPath(
+					t,
+					proxyURL,
+					"",
+					path,
+					http.StatusOK,
+					expectedMagicNumber,
+					nil,
+					consts.IngressWait,
+					consts.WaitTick,
+				)
+			}
+
+			expectContentType(jpegIngressPath, testconsts.JPEGMagicNumber)
+			expectContentType(pngIngressPath, testconsts.PNGMagicNumber)
```

Comment:
```suggestion
			expectContent(jpegIngressPath, testconsts.JPEGMagicNumber)
			expectContent(pngIngressPath, testconsts.PNGMagicNumber)
```
nit: I think suffix `Type` is redundant. WDYT?